include/sys/zio.h

```diff
@@ -223,6 +223,8 @@ typedef uint64_t zio_flag_t;
 #define	ZIO_FLAG_REEXECUTED	(1ULL << 29)
 #define	ZIO_FLAG_DELEGATED	(1ULL << 30)
 
+#define	ZIO_FLAG_ALLOCATOR	(1ULL << 31)
```

Comment:
We could use io_allocator == -1 to indicate its presence/absence.

Reply:
sounds good, i will update.

Reply:
updated, thanks